jupyter_book/07-preprocessing-steps/preprocessing-steps.md

```diff
@@ -0,0 +1,17 @@
+# Preprocessing steps
+
+Before inputting the data into the satay pipeline, it is necessary to preprocess the data that comes from the sequencing company. 
+
+The pipeline understands single end sequencing data, in which the data from every digestion is mixed together. You dont need to analyze them separately. 
+
+## What we do if the sequencing data is paired end: 
```

Comment:
Maybe mention here that when the data is paired end, only one of the pairs will map to the transposon insertion site (the end that has been sequenced from the sequencing primer), while the other end will map back to a location arbitrarily far upstream or downstream of the insertion site (depends on where the restriction site is)

Reply:
ok!